src/wallet.js

```diff
@@ -23,7 +23,11 @@ MyWallet.ws = new BlockchainSocket();
 
 // used locally and overridden in iOS
 MyWallet.socketConnect = function () {
-  MyWallet.ws.connect(onOpen, onMessage, onClose);
+  let socket = MyWallet.ws;
+  socket.connect();
```

Comment:
Not sure about this, but shouldn't the handlers be set up before connecting?

Reply:
I don't think it necessarily matters, because the handlers will only ever be triggered asynchronously. If the handlers were added in a different execution cycle, then it may cause problems. I'll move them to before the call to `connect` anyway to prevent confusion.